Create pagos_sin_cvv.es.md (#231)
mercadopago/devsite-docs: guides/subscriptions/pagos_sin_cvv.es.md

> El Vendedor deberá cumplir con las políticas de integración de Mercado Pago,
>
> A. El Vendedor deberá comunicar en forma clara e inequívoca a su base de usuarios o clientes que la plataforma de pago en su sitio web es provista por Mercado Pago, y los plazos o fechas y los montos de los pagos recurrentes.
>
> B. En caso de que usuarios o clientes existentes del Vendedor estuvieran siendo migrados a la plataforma de Pagos Recurrentes de Mercado Pago, el Vendedor deberá comunicarlo por escrito indicando que Mercado Pago procesará los pagos, informando que en el resumen verá el cargo como Mercado Pago/Mercado Libre” (*).
>
> C. Pre-Approval solo está disponible a través del checkout personalizado de Mercado Pago, es decir, vía la utilización de nuestras API’s.
>
>(*) NOTA: En el caso de tarjetas de crédito Master y Amex, en la tarjeta de crédito aparecerá como: “MERPAG*<brand_name>”. Por lo que para estos medios de pago podrán comunicar: “En tu resumen verás el cargo como MERPAG*<brand_name>” donde <Brand_name> se configura desde la cuenta de Mercado Pago del vendedor: Menu -> Configuracion > Nombre de mi negocio.

> Con los pagos sin cvv, se podrán realizar cobros recurrentes con Mercado Pago teniendo la libertad de amoldar la solución de la forma más óptima para tu negocio

Reviewer: ```suggestion
Con los pagos sin cvv, se podrán realizar cobros recurrentes con Mercado Pago teniendo la libertad de amoldar la solución de la forma más óptima para tu negocio.
```

## Crea una aplicación

> Para crear una aplicación tenes que ingresar con tu cuenta de Mercado Pago a: https://applications.mercadopago.com/ Para poder operar pagos sin cvv la aplicación deberá ser habilitada para tal fin. Estos permisos son asignados desde Mercado Pago, por lo que deberás enviarnos el APP ID obtenido para poder realizar la configuración correspondiente.

Reviewer: ```suggestion
Para crear una aplicación tienes que ingresar con tu cuenta de Mercado Pago a: [https://applications.mercadopago.com/](https://applications.mercadopago.com/), para poder operar pagos sin cvv la aplicación deberá ser habilitada para tal fin. Estos permisos son asignados desde Mercado Pago, por lo que deberás enviarnos el APP ID obtenido para poder realizar la configuración correspondiente.
```

## Cobrá el primer pago

> Para la primer transacción siempre se tendrán que solicitar los datos de la tarjeta, y cursar el pago con código de seguridad. Se podrá hacer siguiendo los pasos de nuestra integración por [API](https://www.mercadopago.com.co/developers/es/guides/payments/api/receiving-payment-by-card/).

Reviewer: ```suggestion
Para la primer transacción siempre se tendrán que solicitar los datos de la tarjeta, y cursar el pago con código de seguridad. Se podrá hacer siguiendo los pasos de nuestra integración por [API](/guides/payments/api/receiving-payment-by-card/).
```

## Crea un customer y asociale la tarjeta utilizada

> Una vez cursado el primer pago, y habiendote asegurado que la tarjeta es válida, crea un Cliente que estará asociado a tu cuenta y asociale una tarjeta. Podrás hacer esto siguiendo el paso a paso indicando en nuestra integración de [Usuarios y Tarjetas](https://www.mercadopago.com.co/developers/es/guides/payments/api/customers-and-cards/)

Reviewer: ```suggestion
Una vez cursado el primer pago, y habiéndote asegurado que la tarjeta es válida, crea un Cliente que estará asociado a tu cuenta y asóciale una tarjeta. Podrás hacer esto siguiendo el paso a paso indicando en nuestra integración de [Usuarios y Tarjetas](/guides/payments/api/customers-and-cards/)
```

> ## Cobrá de forma recurrente a tus clientes

Reviewer: ```suggestion
## Cobra de forma recurrente a tus clientes
```

> ### Obtené el customer guardado

Reviewer: ```suggestion
### Obtén el customer guardado
```

Para conocer los datos de tu cliente, podrás obtenerlo de la siguiente forma:

[[[
```php
<?php
require_once ('mercadopago.php'); $mp = new MP ("TEST-8770266498150001-062911-
821263869b3801c4f007924913b979ea__LB_LD__-
186597721"); $filters = array ("email" => "your.payer@email"); $customer = $mp->get
("/v1/customers/search", $filters);
print_r ($customer);
?>
```
]]]

### Otené la tarjeta asociada a tu cliente

Una vez hayas obtenido el id de tu cliente, puedes buscar la tarjeta de la siguiente forma:

[[[
```php
<?php
require_once ('mercadopago.php');
$mp = new MP ("TEST-8770266498150001-062911-
821263869b3801c4f007924913b979ea__LB_LD__-
186597721");
$cards = $mp->get ("/v1/customers/[CUSTOMER_ID]/cards");
print_r ($cards["response"]);
?>
```
]]]

> ### Tokenizá la tarjeta con el card_id

Reviewer: ```suggestion
### Tokeniza la tarjeta con el card_id
```

[[[
```php
<?php
require_once ('mercadopago.php');
$mp = new MP ("TEST-8770266498150001-062911-
821263869b3801c4f007924913b979ea__LB_LD__-
186597721");
$card_token = $mp->post ("/v1/card_tokens", array("card_id" => "cardId"));
print_r ($card_token);
?>

```
]]]

> ### Realizá el cobro:

Reviewer: ```suggestion
### Realiza el cobro:
```
Se suele evitar imperativo para adaptarnos mejor a MLM y MCO

> Cómo estás usando un token creado con el card_id, deberás realizar el posteo del pago indicando el id del customer asociado a la tarejta:

Reviewer: ```suggestion
Al estar usando un token creado con el card_id, deberás realizar el posteo del pago indicando el id del customer asociado a la tarjeta:
```

[[[
```php
<?php
require_once ('mercadopago.php');
$mp = new MP('TEST-8770266498150001-062911-
821263869b3801c4f007924913b979ea__LB_LD__-186597721');
$payment_data = array(
"transaction_amount'" => 100,
"token'" => "ff8080814c11e237014c1ff593b57b4d",
"description'" => "Title of what you are paying for",
"installments'" => 1,
"payer'" => array (
"id" => "12345678"
)
);
$payment = $mp->post("/v1/payments", $payment_data);
?>
```
]]]

## Escuha notificaciones de los pagos

> Cada que vez que se curse un pago y haya una novedad sobre el pago, Mercado Pago te enviará una notificación para que puedas actualizar tus sistemas. Podrás ver el paso a paso en nuestra sección de [notificaciones](https://www.mercadopago.com.co/developers/es/guides/notifications/webhooks/)

Reviewer: ```suggestion
Cada que vez que se curse un pago y haya una novedad sobre el pago, Mercado Pago te enviará una notificación para que puedas actualizar tus sistemas. Podrás ver el paso a paso en nuestra sección de [notificaciones](/guides/notifications/webhooks/)
```

## Reintentos

Si el pago sin cvv es rechazado, te recomendamos que sigas una lógica de reintentos según el estado del rechazo. Por ejemplo, si el pago fue rechazado por tarjeta vencida no tiene sentido que se haga un reintento. Se le deberá solicitar al cliente que informa otra tarjeta para cursar los cobros siguientes. En caso que el rechazo sea por fondos insuficientes, tiene sentido que se haga una lógica de reintentos. 

> Podrás ver todos los rechazos [aquí](https://www.mercadopago.com.co/developers/es/guides/payments/api/handling-responses/)

Reviewer: ```suggestion
Podrás ver todos los rechazos [aquí](/guides/payments/api/handling-responses/)
```
